packages/valory/skills/decision_maker_abci/policy.py

```diff
@@ -137,12 +137,14 @@ def update_weighted_accuracy(self) -> None:
             for tool, acc_info in self.accuracy_store.items()
         }
 
-    def select_tool(self, randomness: RandomnessType) -> Optional[str]:
+    def select_tool(self, randomness: RandomnessType = None) -> Optional[str]:
```

Comment:
you do not need to change the calls to this function so that only once the seed is past as a parameter?